src/Stratis.Bitcoin.Features.SmartContracts/Stratis.Bitcoin.Features.SmartContracts.csproj

```diff
@@ -5,6 +5,7 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="CSharpFunctionalExtensions" Version="1.10.0" />
```

Comment:
Whats this?

Reply:
This was missing from the Stratis.Bitcoin.Features.SmartContracts.csproj file and stopped me from building this project.

Here is the original file from the initial SC commit - https://github.com/stratisproject/StratisBitcoinFullNode/commit/bf7c94d0d9959a6be79910ff072d7e805f9fb01e#diff-e1a79fe58674a2072ea55f05485e7987R7.